data/org.freedesktop.UDisks2.xml

```diff
@@ -2971,6 +2971,21 @@
       <arg name="options" direction="in" type="a{sv}"/>
     </method>
 
+
+    <!--
+        Convert:
+        @target_version: The LUKS version to convert to. Either 'luks1' or 'luks2'.
+        @options: Options (currently unused except for <link linkend="udisks-std-options">standard options</link>).
+        @since: 2.11.0
+
+        Converts the encrypted device to a different version of LUKS. Other encryption formats are not supported.
+        The device must be offline.
```

Comment:
I'd say "must be locked" here, we use Lock/Unlock terminology in udisks so it would be clearer.